server/api/api/utils.py

```diff
@@ -1396,8 +1396,6 @@ async def delete_function(
             if isinstance(result, tuple):
                 nuclio_name, error_message = result
                 if error_message:
-                    failed_requests.append(
-                        f"Failed to delete nuclio function {nuclio_name}: {error_message}"
-                    )
+                    failed_requests.append(error_message)
```

Comment:
See [my comment on JIRA](https://iguazio.atlassian.net/browse/ML-6552?focusedCommentId=151269), and try to extract the relevant info from the exception